### PR TITLE
python3.pkgs.woodblock: init at 0.1.7

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120003` at d862230a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.multimethod</li>
    <li>python38Packages.woodblock</li>
    <li>python39Packages.multimethod</li>
    <li>python39Packages.woodblock</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/multimethod/default.nix:22:0:
```
   |
22 |     description = "Multiple argument dispatching";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/woodblock/default.nix:24:0:
```
   |
24 |     description = "A framework to generate file carving test data";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/multimethod/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pytest-cov
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/multimethod/default.nix:7]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/multimethod/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  pname = "multimethod";
+  version = "1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b9c6f85ecf187f14a3951fff319643e1fac3086d757dec64f2469e1fd136b65d";
+  };
+
+  checkInputs = [
+    pytest
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/multimethod/default.nix:17]:
```suggestion
    pytestCheckHook
```

File: pkgs/development/python-modules/multimethod/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/multimethod/default.nix:4]:
```suggestion
, pytestCheckHook
```

File: pkgs/development/python-modules/multimethod/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b9c6f85ecf187f14a3951fff319643e1fac3086d757dec64f2469e1fd136b65d";
+  };
+
+  checkInputs = [
+    pytest
+    pytest-cov
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/multimethod/default.nix:20]:
```suggestion

  pythonImportsCheck = [ "multimethod" ];

```

File: pkgs/development/python-modules/woodblock/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, multimethod
+, numpy
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/woodblock/default.nix:8]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/woodblock/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    inherit pname version;
+    sha256 = "c0347ece920b7009d94551983a01f42db02920ca8d7b0ff36d24a337e2c937f7";
+  };
+
+  propagatedBuildInputs = [
+    click
+    multimethod
+    numpy
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/woodblock/default.nix:22]: Missing checkPhase and pythonImportsCheck.

**Pull request merged**